applications/DamApplication/python_scripts/impose_face_heat_flux_process.py

```diff
@@ -18,18 +18,16 @@ def __init__(self, Model, settings ):
 
         ## This process assign and uniform heat flux
         if "Uniform" in settings["model_part_name"].GetString():
-            t_uniform = Parameters("{}")
-            t_uniform.AddValue("model_part_name",settings["model_part_name"])
-            t_uniform.AddValue("mesh_id",settings["mesh_id"])
-            t_uniform.AddEmptyValue("is_fixed").SetBool(False)
-            t_uniform.AddValue("variable_name",settings["variable_name"])
-            t_uniform.AddValue("value",settings["value"])
-
             if settings["table"].GetInt() == 0:
+                t_uniform = Parameters("{}")
+                t_uniform.AddValue("model_part_name",settings["model_part_name"])
+                t_uniform.AddValue("mesh_id",settings["mesh_id"])
+                t_uniform.AddEmptyValue("is_fixed").SetBool(False)
+                t_uniform.AddValue("variable_name",settings["variable_name"])
+                t_uniform.AddValue("value",settings["value"])
                 self.components_process_list.append(ApplyConstantScalarValueProcess(model_part, t_uniform))
             else:
-                t_uniform.AddValue("table",settings["table"])
-                self.components_process_list.append(ApplyDoubleTableProcess(model_part, t_uniform))
+                self.components_process_list.append(DamFixTemperatureConditionProcess(model_part, settings))
```

Comment:
Why do you define DamFixTemperatureConditionProcess in impose_face_heat_flux_process.py ? Probably I'm forgetting something...

Reply:
Hi @ipouplana! As far as I am concerned, this is necessary because although using heat flux you need to specify the ambient temperature.

Reply:
Ok, thanks!